Jenkinsfile

```diff
@@ -442,74 +449,77 @@ pipeline {
+    //   when {
+    //     anyOf{
+    //       branch 'main'
+    //       changeRequest target: 'main'
+    //     }
+    //   }
+    //   failFast true
+    //   steps{
+    //     sh 'cd examples/nlp/text_classification && \
+    //     python text_classification_with_bert.py \
+    //     exp_manager.create_checkpoint_callback=false \
+    //     exp_manager.exp_dir=exp_mp_2_megatron_bert \
+    //     trainer.gpus=[0,1] \
+    //     trainer.num_nodes=1 \
+    //     trainer.precision=16 \
+    //     ~trainer.amp_level \
+    //     +trainer.replace_sampler_ddp=false \
+    //     +trainer.fast_dev_run=true \
+    //     model.dataset.num_classes=6 \
+    //     model.train_ds.file_path=/home/TestData/nlp/retail_text_classification/train.tsv \
+    //     model.train_ds.batch_size=4 \
+    //     model.language_model.pretrained_model_name=megatron-bert-uncased \
+    //     model.language_model.config_file=/home/TestData/nlp/mp_2_bert_toy/config.json \
+    //     model.language_model.lm_checkpoint=/home/TestData/nlp/mp_2_bert_toy/iter_2000000 \
+    //     '
+    //     sh 'rm -rf examples/nlp/text_classification/exp_mp_2_megatron_bert'
+    //   }
+    // }
 
-    stage('L2: Parallel NLP Examples 2') {
-      when {
-        anyOf{
-          branch 'main'
-          changeRequest target: 'main'
-        }
-      }
-      failFast true
-      parallel {
-        stage ('NER finetuning from pretrained Test') {
-          steps {
-            sh 'cd examples/nlp/token_classification && \
-            python token_classification.py \
-            pretrained_model=NERModel \
-            model.dataset.data_dir=/home/TestData/nlp/ner/ \
-            model.train_ds.batch_size=2 \
-            model.dataset.use_cache=false \
-            trainer.gpus=[0] \
-            +trainer.fast_dev_run=true \
-            exp_manager.exp_dir=examples/nlp/token_classification/ner_from_pretrained'
-            sh 'rm -rf examples/nlp/token_classification/ner_from_pretrained'
-          }
-        }
-        stage ('Punctuation and capitalization finetuning from pretrained test') {
-          steps {
-            sh 'cd examples/nlp/token_classification && \
-            python punctuation_capitalization.py \
-            pretrained_model=Punctuation_Capitalization_with_BERT \
-            model.dataset.data_dir=/home/TestData/nlp/token_classification_punctuation/ \
-            trainer.gpus=[1] \
-            +trainer.fast_dev_run=true \
-            model.dataset.use_cache=false \
-            exp_manager.exp_dir=examples/nlp/token_classification/pc_from_pretrained'
-            sh 'rm -rf examples/nlp/token_classification/pc_from_pretrained'
-          }
-        }
-      }
-    }
+    // TODO: Pytorch Lightning has some issues with restoring Metric classes, asked on the lightning slack if they can
+    // provide a simple solution.
+    // stage('L2: Parallel NLP Examples 2') {
```

Comment:
'NER finetuning from pretrained Test' is currently blocked by https://github.com/PyTorchLightning/pytorch-lightning/pull/4195

Reply:
Think you might be able to use `Lightning.load_from_checkpoint(..., strict=False)` to avoid the key mismatch. (Uses [`load_state_dict(..., strict=False)`](https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module.load_state_dict) under the hood.